src/rf/apps/core/models.py

```diff
@@ -133,6 +135,30 @@ class Meta:
     thumb_large_key = CharField(max_length=255, blank=True, default='',
                                 help_text='S3 key for large thumbnail')
 
+    def retry_possible(self):
+        """
+        Returns true if it is possible to retry processing a layer.
+        """
+        return self.status == enums.STATUS_FAILED
```

Comment:
I tend to agree with you, but @kdeloach wanted to be able to retry on any failure. I think the argument was that it was less complicated and could be useful during development.